riocli/jsonschema/schemas/package-schema.yaml

```
@@ -137,6 +140,10 @@ definitions:
           memory:
             type: number
             minimum: 0
+      livenessProbe:
+        type: object
+        items:
+          "$ref": "#definations/livenessProbe"
```

Please correct this.
```suggestion
      livenessProbe:
        type: object
        "$ref": "#/definitions/livenessProbe"
```